packages/next/build/webpack/plugins/wellknown-errors-plugin/parseNotFoundError.ts

```diff
@@ -1,9 +1,51 @@
 import Chalk from 'chalk'
 import { SimpleWebpackError } from './simpleWebpackError'
 import { createOriginalStackFrame } from '@next/react-dev-overlay/lib/middleware'
+import { isWebpack5 } from 'next/dist/compiled/webpack/webpack'
+import path from 'path'
 
 const chalk = new Chalk.constructor({ enabled: true })
 
+// Based on https://github.com/webpack/webpack/blob/main/lib/stats/DefaultStatsFactoryPlugin.js#L422-L431
```

Comment:
```suggestion
// Based on https://github.com/webpack/webpack/blob/fcdd04a833943394bbb0a9eeb54a962a24cc7e41/lib/stats/DefaultStatsFactoryPlugin.js#L422-L431
```

How about including the sha so this link won't 404 if the file is moved